### PR TITLE
Change problem.txt to problem.md for all problems.

### PR DESCRIPTION
**Pull request by jordanSu**
With problem.txt, problem description in every problem are shown as grey-scale color without syntax highlighting or emphasis highlighting, which may make it hard to read.

This changes update all problem.txt to problem.md and
- Specify the language used (ex. js, sh) for code syntax highlighting.
- Adjust paragraph format to make it easy to read.

### CODE REVIEW

File: problems/beep_boop/problem.md [hunk 1]
```diff
@@ -1,8 +1,10 @@
-Make a new directory for your stream-adventure solutions (mkdir stream-adventure and enter it cd ./stream-adventure)
+Make a new directory for your stream-adventure solutions (`mkdir stream-adventure` and enter it `cd ./stream-adventure`)
 Create a new file called beep_boop.js that uses console.log to output "beep boop".
 
 To verify your program has the expected output, run:
 
-  stream-adventure verify beep_boop.js
+```sh
+$ {appname} verify baby-steps.js
```
Comment (ccarruitero) [inline problems/beep_boop/problem.md:7]: should be `beep_boop.js` here

**Comment by ccarruitero** [comment]
Thanks @jordanSu 

Seems good, just the file name changed.

**Comment by ccarruitero** [comment]
@jordanSu I corrected the file name and rebased your changes into master in https://github.com/workshopper/stream-adventure/commit/6704af081794bcec13651a366a9d38e34e53357f

Thanks again for your contribution

**Comment by jordanSu** [comment]
@ccarruitero thanks for helping me correct that.

**Pull request closed without merging**